src/FakeItEasy/Core/DefaultFakeManagerAccessor.cs

```diff
@@ -26,7 +26,7 @@ public FakeManager GetFakeManager(object proxy)
 
             if (result == null)
             {
-                throw new ArgumentException("The specified object is not recognized as a fake object.");
+                throw new ArgumentException(string.Format(ExceptionMessages.NotRecognizedAsFake, proxy.ToString(), proxy.GetType().FullNameCSharp()));

```

Comment:
Yes that's fine. I'm not very familiar with rebasing, can you provide a bit more detail of how I might perform this action?